spec/system/admin/reports_spec.rb

```diff
@@ -113,6 +109,43 @@
           .to raise_error OpenURI::HTTPError, "404 Not Found"
       end
     end
+
+    pending "allows the report to finish before the loading screen is rendered" do
+      login_as_admin
```

Comment:
@mkllnk  We could probably mitigate that by having the loading spinner and the report being rendered in two different div? Then the spinner wouldn't override the existing report ? It's not ideal to show a spinner when the report is already there but I think it's a better alternative ? What do you think ?